processors/sshd/sshdprocessor.go

```diff
@@ -321,6 +326,77 @@ func processAcceptPublicKeyEntry(config *SshdProcessorer) error {
 	}
 }
 
+func processAcceptedPasswordEntry(config *SshdProcessorer) error {
+	pid, err := strconv.Atoi(config.pid)
+	if err != nil {
+		logger.Errorf("failed to convert pid string to int ('%s') - %s",
+			config.pid, err)
+		return nil
+	}
+
+	matches := passwordLoginRE.FindStringSubmatch(config.logEntry)
+	if matches == nil {
+		logger.Infoln("got processAcceptedPasswordEntry log with no string sub-matches")
+		return nil
+	}
+
+	var username string
+	usernameIdx := passwordLoginRE.SubexpIndex(idxLoginUserName)
+	if usernameIdx > -1 {
+		username = matches[usernameIdx]
+	}
+
+	var source string
+	sourceIdx := passwordLoginRE.SubexpIndex(idxLoginSource)
+	if sourceIdx > -1 {
+		source = matches[sourceIdx]
+	}
+
+	var port string
+	portIdx := passwordLoginRE.SubexpIndex(idxLoginPort)
+	if portIdx > -1 {
+		port = matches[portIdx]
+	}
+
+	evt := auditevent.NewAuditEvent(
+		common.ActionLoginIdentifier,
+		auditevent.EventSource{
+			Type:  "IP",
+			Value: source,
+			Extra: map[string]any{
+				"port": port,
+			},
+		},
+		auditevent.OutcomeSucceeded,
+		map[string]string{
+			"loggedAs": username,
+			"userID":   common.UnknownUser,
+			"pid":      config.pid,
+		},
+		"sshd",
+	).WithTarget(map[string]string{
+		"host":       config.nodeName,
+		"machine-id": config.machineID,
+	})
+
+	evt.LoggedAt = config.when
+
+	if err := config.eventW.Write(evt); err != nil {
+		return fmt.Errorf("failed to write event: %w", err)
+	}
+
+	select {
+	case <-config.ctx.Done():
+		return nil
```

Comment:
Just noticed this - you will want to return `ctx.Err()` here instead of `nil`. (I know this probably came from some other code - but we should change that code to do the same thing)